### ISSUE TITLE
SW dot

### ISSUE THREAD

**Issue by globalgame** [WarEmu/WarBugs#19101]
Broadhead arrow damage is insane 1 tick of the dot is more than 1k damage 
I do not know what developers did with SW but i did not play for a year and it looks crazy
Just 1 dot can do more than 5k damage
![WAR-64 2021-06-08 14-01-40-84](https://user-images.githubusercontent.com/24555848/121168894-1d5b4800-c85c-11eb-8674-6fd49398bab2.png)

**Comment by lilimx** [comment]
Hey,

thank you for your ticket. But I need more information about the circumstences. I miss the timestamps on your combat log. Where was the fight? What's your lvl/rr?

**Comment by IsMiseFionnbharr** [comment]
Just FYI, you were hit by Burn Armour by the WH before the massive BHA ticks, which reduces your armour by 1k, as well as the SW likely running Powerful Draw, which means your armour was probably reduced by ~50% or more. 3 stack BHA crits 1k on pve targets, so this seems expected behaviour

**Comment by Grock-r** [comment]
Burn armor wouldn't change the total dmg, only the ratio between dmg taken and dmg mitigated.

Skirmisher SW was reworked last year and BHA now stacks up to 3 times, keep in mind they also have Vengeance of Nagarythe 25% dmg buff and a toggle ability that increases crit dmg by 25%, so the final damage can get pretty massive 

My 22r SW's BHA does 141 dmg per tick per stack, up to 423 at 3 stacks. 
If i enable VoN and Steady Aim the critical damage goes up to 930~ (the highest i got so far is 946)
![BHAdmg1](https://puu.sh/HNhWK/bec6ee7d93.jpg)
![BHAdmg2](http://puu.sh/HNi0F/14e17c47ba.jpg)

And thats just on a little 22rank SW with lvl20 weapon and only 491 BS. 
If you take a 80+ Sovereign SW their BHA damage could probably be double of what i have.

So in short: damage is bonkers, but its most likely not a bug.

**Comment by lilimx** [comment]
Hey,

yeah, the dmg seems not buged or something. So I will close this one.
